src/Service/Core.php

```diff
@@ -39,7 +40,7 @@ class Core {
    *   Module Handler Interface.
    */
   public function __construct(LoggerChannelFactoryInterface $loggerFactory,
-                              ModuleHandlerInterface $moduleHandler) {
+    ModuleHandlerInterface $moduleHandler) {
```

Comment:
is this a personal tabbing pref or something relating to a coding standard?

Reply:
auto-formatting, probably